### PR TITLE
First

### PR DESCRIPTION
**Pull request by nick-benoit14**
https://www.pivotaltracker.com/story/show/183037093

### CODE REVIEW

File: lib/atomic_lti_1v1/lti_1v1.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 module AtomicLti1v1
   class Lti1v1
     def self.is_lti_1v1?(request)
-      request.params['oauth_consumer_key'].present?
+      contains_oauth_consumer_key = request.params['oauth_consumer_key'].present? 
+      path_matches = request.env["PATH_INFO"].match(AtomicLti1v1.lti_launch_path_regex)&.size == 1
```
Comment (mpetrowi) [inline lib/atomic_lti_1v1/lti_1v1.rb:5]: I wonder if we should allow an array of launch paths. There could be multiple

**Review by mpetrowi — APPROVED** [review]

File: lib/atomic_lti_1v1/lti_1v1.rb (hunk 1 above)
Comment (nick-benoit14) [reply lib/atomic_lti_1v1/lti_1v1.rb:5]: Good call

Comment (nick-benoit14) [reply lib/atomic_lti_1v1/lti_1v1.rb:5]: Though this is a regex so I guess you could already do that. Maybe we should say it shouldn't be a regex and instead be a list of paths?

Comment (nick-benoit14) [reply lib/atomic_lti_1v1/lti_1v1.rb:5]: Do list of paths that are interpreted as path prefixes

**Comment by mpetrowi** [comment]
@nick-benoit14  I wonder if we should close this.  The regexp part isn't needed anymore, and I'm not sure we need the exception stuff at this point.

**Pull request closed without merging**